frontend/test/metabase/scenarios/admin/datamodel/segments.cy.spec.js

```diff
@@ -105,7 +105,7 @@ describe("scenarios > admin > datamodel > segments", () => {
       cy.findAllByText("Filter")
         .first()
         .click();
-      sidebar().within(() => {
+      cy.findByTestId("sidebar-right").within(() => {
```

Comment:
Same thought about using custom aria `role` instead of `data-testid` comes to mind.

Reply:
I left it with the test id because there are two sidebars on the page